hpedockerplugin/hpe_storage_api.py

```diff
@@ -44,6 +44,7 @@
 DEFAULT_PROV = "thin"
 DEFAULT_FLASH_CACHE = None
 DEFAULT_MOUNT_VOLUME = "True"
+DEFAULT_COMPRESSION_VAL = None

```

Comment:
default value for compression can be "False"

Reply:
Hi William,
Thanks for the review.
Here i have purposely kept the value as None. As user may choose not to pass compression option (which is a valid case) it should work fine.
The valid flags are true/false. In case if user is not using 3.3.1 system, code flow should not pass the compression flag in request body. To differentiate between boolen and None value i have this value as None. 
Another use case is, user has 3.3.1 OS and he opts to choose compression option as Flase, it is suppose to work.